src/main/java/Duke.java

```diff
@@ -1,10 +1,402 @@
+            addEvent(input);
+        } else if (input.startsWith("deadline ")) {
+            //Adds new DeadlineTask
+            addDeadline(input);
+        } else if (input.startsWith("delete ")) {
+            //Delete specified tasks
+            deleteTasks(input);
+        } else {
+            //Prints error (unrecognized command)
+            System.out.println(MESSAGE_UNRECOGNIZED_COMMAND);
+        }
+    }
+
+
+    /**
+     * Ends while-loop to exit program.
+     */
+    private static void endProgram() {
+        System.out.println(MESSAGE_BYE);
+        try {
+            saveFile();
+        } catch (Exception e) {
+            System.out.println(MESSAGE_SAVE_ERROR);
+        }
+        endProgramNow = true;
+    }
+
+
+    /**
+     * Prints list of task name.
+     */
+    private static void printList() {
+        System.out.println(LINE);
+        if (taskCount == 0) {
+            System.out.println(MESSAGE_EMPTY_LIST);
+            return;
+        }
+        for (int i = 0; i < taskCount; i++) {
+            System.out.print((i + 1) + ".");
+            tasks.get(i).printStatus();
+            System.out.println();
+        }
+        System.out.println(LINE);
+    }
+
+
+    /**
+     * Marks specified task as done.
+     * If position is invalid, print error message.
+     *
+     * @param input Input value by user.
+     */
+    private static void setDoneStatus(String input, Boolean isDone) {
+        try {
+            int position = Integer.parseInt(input.split(" ")[1]) - 1;
+            if (position >= taskCount || position < 0) {
+                //Out of bounds
+                throw new IndexOutOfBoundsException();
+            }
+            tasks.get(position).setDone(isDone);
+            if (isDone) {
+                System.out.println(MESSAGE_MARK_DONE);
+            } else {
+                System.out.println(MESSAGE_MARK_UNDONE);
+            }
+            tasks.get(position).printStatus();
+            System.out.println("\n" + LINE);
+        } catch (Exception e) {
+            System.out.println(MESSAGE_UNRECOGNIZED_COMMAND);
+        }
+    }
+
+
+    /**
+     * Creates new ToDoTask.
+     *
+     * @param input Input value by user.
+     */
+    private static void addToDo(String input) {
+        try {
+            if (taskCount >= MAX_SIZE) {
+                //Array full
+                throw new ListFullException();
+            }
+            input = input.substring(LENGTH_OF_WORD_TODO + 1);
+            tasks.add(new ToDoTask(input));
+            taskCount++;
+            printAddedContent(input);
+        } catch (ListFullException e) {
+            System.out.println(MESSAGE_LIST_FULL);
+        }
+    }
+
+
+    /**
+     * Creates new EventTask.
+     * Prints error message if invalid.
+     *
+     * @param input Input value by user.
+     */
+    private static void addEvent(String input) {
+        try {
+            if (!input.contains(" /at ")) {
```

Comment:
You might want to consider declaring " /at " as a constant for better reusability.